webapp/stores/team_store.jsx

```diff
@@ -154,8 +155,9 @@ class TeamStoreClass extends EventEmitter {
             Utils = require('utils/utils.jsx'); //eslint-disable-line global-require
```

Comment:
One other thing, Ùtils` is not declared anywhere, so it will be registered as global.